src/screens/AccountSecurityScreen/index.js

```diff
@@ -1,26 +1,37 @@
 import React, {useState} from 'react';
+import {useFocusEffect} from '@react-navigation/native';
 import {FormattedMessage} from 'react-intl';
 
+import useQueryWithAuth from '@/hooks/useQueryWithAuth';
 import ModalContainer from '@/components/ModalContainer';
 import ListOption from '@/components/ListOption';
 import SpecialListOption from '@/components/SpecialListOption';
 import Switch from '@/components/Switch';
+import {GET_USER_SECURITY_SETTINGS} from '@/api/data';
 
 import {Container} from './style';
 
 const AccountSecurityScreen = ({navigation}) => {
-  const [isPinToggled, setIsPinToggled] = useState(false); // from api later
   const [isFaceIdToggled, setIsFaceIdToggled] = useState(false); // from api later
+  const {data, refetch} = useQueryWithAuth(GET_USER_SECURITY_SETTINGS, {
+    fetchPolicy: 'network-only',
+  });
+  const isPinSet = data?.userProfile?.isPasscodeSet;
+
+  useFocusEffect(() => {
+    refetch();
+  }, [navigation]);
```

Comment:
`useFocusEffect` accepts one param only. Please change it to 
```suggestion
  useFocusEffect(refetch);
```

Reply:
https://reactnavigation.org/docs/use-focus-effect/
it should be wrap in useCallback?

```
useFocusEffect(
    useCallback(() => {
      refetch();
    }, [refetch]),
  );
```

Reply:
For this case it is not necessary. Just use `useFocusEffect(refetch);`. Think about why you don't need it. I will ask you later